apple.china.conf

```diff
@@ -3,9 +3,11 @@ server=/a2.mzstatic.com/114.114.114.114
 server=/a3.mzstatic.com/114.114.114.114
 server=/a4.mzstatic.com/114.114.114.114
 server=/a5.mzstatic.com/114.114.114.114
+server=/aaplimg.com/114.114.114.114
 server=/adcdownload.apple.com/114.114.114.114
 server=/appldnld.apple.com/114.114.114.114
 server=/apps.mzstatic.com/114.114.114.114
+server=/bag.itunes.apple.com/114.114.114.114
```

Comment:
`itunes.apple.com` is already present, there is no need to add a subdomain.

Reply:
Agree!